test/server-test.js

```diff
@@ -123,6 +123,30 @@ describe('Server', function(){
         })
       })
     })
+
+    describe("PUT /foods/:id", function(){
+
+      beforeEach(function(done){
+        Foods.createFoods("pizza", 155).then(function () { done() });
+      })
+
+      afterEach(function(done){
+        Foods.resetFoods().then(function () { done() })
+      })
+
+      it("can update existing food", function(done){
+        var food = {name: "cheese pizza"}
+
+        this.request.put("api/v1/foods/1", {form: food}, function(error, response){
```

Comment:
this `put` request should trigger the console.log in the above file. It's not, for some reason.

Any ideas on how to get that console.log (in server.js:52) to run?

@DBULL7 ^^